.github/actions/spelling/patterns.txt

```diff
@@ -15,3 +15,5 @@ data:[a-zA-Z=;,/0-9+-]+
 \b[0-9a-f]{40}\b
 \b([A-Za-z])\1{3,}\b
 El proyecto .* diferentes
+# Package family names
+[-.A-Za-z0-9]+_[a-z0-9]{13}
```

Comment:
This matches:
https://github.com/lechacon/winget-cli/blob/bb9016b37277bd43d4c7de6fc1d07cd5d5f85a5f/src/AppInstallerCLITests/Settings.cpp#L87
So instead of seeing:
```
    StreamDefinition name{ Type::Secure, "secure_nonexistentsetting" };
```
The spell checker saw:
```
    StreamDefinition name{ Type::Secure, "tting" };
```

Reply:
This fixes the specific complaint:
```suggestion
\b[-.A-Za-z0-9]+_[a-z0-9]{13}\b
```
(The `\b` at the beginning isn't important, the one at the end is.)

I can't promise the spell checker won't complain about something else. (I'm out of time for today.)

Reply:
Thank you! I couldn't have figured that out myself.